New Verne

Albens opened the pull request:
A new, lovingly redone Verne.
Intended to build upon the lessons learned since Torch launch, this map is designed to drive interaction while providing overmap agency for the Verne's crew.
Features:
4 Crew. (Ie, high-count map)
1 Engineer, an employee of CTI.
1 Pilot, an employee of CTI.
2 Undergraduate Xenosci researchers.

Some temporary lore has been added. 
This is ready for review.
<details>
A shuttle of their own, the SRV Venerable Catfish.
A rough goal or objective will be provided- they are a Xeno-science Survey team, out to do Undergraduate things.
Lore is not added a this time but will be at a Soon(tm) date.

This map is also a showcase for experimental features- Multi-deck SM Chambers, External Atmospheric pods, and a Hangar that, one day, may allow the Shuttle to Dock in a sensible manner.

Intended lore:
The SRV Verne, operated by CTI (Ceti Technology Institute), is a semi-autonomous Research Vessel, providing facilities for Undergraduate Xeno-scientists, and an experimental Supermatter research facility.
Captained by two CTI Professors, each holding Co-Consulship on a cryostasis shift system, with teams waking up as required to perform their studies, or to operate the vessel.
Due to University Politics, Backstabbing, and the unusual Captainship system, the Verne has found itself dramatically over-extended, under-staffed, running on 100% automation.

Special notes:
The SM-Research Facility is fully functional, but is intended to be an ornamental piece, as it does not drive interaction to sit on the Verne and play with the SM all day.
The Survey Team lacks a doctor. The scientists will have some latent medical skill, but the Surveyors best avoid danger.
The ship puts up a fine defense, when in a pre-spawn configuration, to deter overmap provocateurs.
Old-verne is gone, but may one day return as something different. Idk.

This map is not freeform. You will be a specific team, you are from a specific school, with specific employment or post-secondary education. That's a bother to some, but I believe the lore as intended is not overly restrictive, as it allows the power gained from being a formal SRV Vessel to be balanced with the enforced lore.
 </details>
With much help from Venku22 and Nerezza, Hubblenaut, and Rowtree.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

SierraKomodo commented on code/modules/power/port_gen.dm:
```diff
@@ -501,6 +501,59 @@
+	name = "nuclear reactor"
+	desc = "ICRER-2, an industrial-yet-compact nuclear fusion reactor powered by sheets of uranium and liquid coolant. Rated for (XXX) kW maximum safe output- exceeding it is likely to result in a radioactive detonation and is not recommended."
+	power_gen = 30000			//Watts output per power_output level
+	icon_state = "potato"
```
Potato?

Hubblenaut replied:
`potato` is the icon_state used for the nuclear reactor

SierraKomodo commented on code/modules/power/port_gen.dm:
```diff
@@ -501,6 +501,59 @@
+		temperature_gain = 60
+		reagents.remove_any(1)
+		if(prob(2))
+			audible_message("<span class='notice'>[src] churns happily</span>")
```
SPAN_NOTICE()

SierraKomodo commented on code/modules/power/port_gen.dm:
```diff
@@ -501,6 +501,59 @@
+		var/obj/item/weapon/reagent_containers/R = O
+		if(R.standard_pour_into(src,user))
+			if(reagents.has_reagent("coolant"))
+				audible_message("<span class='notice'>[src] blips happily</span>")
```
SPAN_NOTICE()

SierraKomodo commented on code/modules/power/port_gen.dm:
```diff
@@ -501,6 +501,59 @@
+				audible_message("<span class='notice'>[src] blips happily</span>")
+				playsound(get_turf(src),'sound/machines/synth_yes.ogg', 50, 0)
+			else
+				audible_message("<span class='warning'>[src] blips in disappointment</span>")
```
SPAN_WARNING()

SierraKomodo commented on maps/away/verne/verne_jobs.dm:
```diff
@@ -0,0 +1,66 @@
+	total_positions = 1
+	outfit_type = /decl/hierarchy/outfit/job/verne/pilot
+	supervisors = "the CTI Professor in Command of the Verne"
+	info = " SRV Verne, as it carried the members of the presitigous Ceti Technical institute in \
```
Extra leading space

SierraKomodo reviewed (changes requested):
Not exhaustive, just a couple things I noticed

Hubblenaut commented on code/modules/power/port_gen.dm:
```diff
@@ -501,6 +501,59 @@
 		return
 	..()
 
+/obj/machinery/power/port_gen/pacman/super/reactor
```
This should be a subtype of `/obj/machinery/power/port_gen/pacman/super/potato`, especially since everything but the description appears to be the same.

Albens commented:
Updated with much help (he literally did it for me <3) from @Hubblenaut .
Addresses @SierraKomodo 's concerns by not copy-pasting code and instead doing it properly as a subtype.

Hubblenaut commented on code/modules/power/port_gen.dm:
```diff
@@ -449,18 +449,19 @@
+    rad_power = 12
+    atom_flags = ATOM_FLAG_OPEN_CONTAINER
+    anchored = 1
+    var/coolant_reagent = "vodka"
```
There seem to have gotten a bunch of unintended spacing changes into the above lines, I assume from copying from Discord.

Albens replied:
Oh god I thought I fixed this.

Hubblenaut commented on code/modules/power/port_gen.dm:
```diff
@@ -501,6 +502,18 @@
+	max_power_output = 8	//The maximum power setting without emagging.
+	temperature_gain = 80	//how much the temperature increases per power output level, in degrees per level
+	max_temperature = 450
+	rad_power = 12
```
The only two values here that are different from the supertype are `desc` and `coolant_reagent`. Unless you plan on changing the others, they should probably be removed.

Albens replied:
I do intend to change the others as per 
> The new reactor needs to be modified and like, updated.

Hubblenaut reviewed (changes requested):
I'm not sure if you planned for the reactor to be for the Verne only or a general addition to the game.
If the latter, I'll note that it won't be able to be researched without a research design added in `design_circuits.dm`.
If it's supposed to be for the Verne only, I believe it should better be moved into `verne.dm`.

Hubblenaut commented on code/modules/power/port_gen.dm:
```diff
@@ -449,18 +449,19 @@
+    rad_power = 12
+    atom_flags = ATOM_FLAG_OPEN_CONTAINER
+    anchored = 1
+    var/coolant_reagent = "vodka"
```
```suggestion
    var/coolant_reagent = /datum/reagent/ethanol/vodka
```
Having had another look at reagent code, this will indeed have to be a type now for it to work, strings are no longer supported by procs like `has_reagent()` and will fail silently.
Likewise, you will need to use `datum/reagent/coolant` instead of `"coolant"` below.

Albens commented:
> I'm not sure if you planned for the reactor to be for the Verne only or a general addition to the game.
> If the latter, I'll note that it won't be able to be researched without a research design added in `design_circuits.dm`.
> If it's supposed to be for the Verne only, I believe it should better be moved into `verne.dm`.

Given the likelyhood of other maps using the potato reactor, such as original verne, a general addition seems more applicable.

I'll add it to `design_circuits.dm` if there's a good argument for the Torch building it, so it'll probably be added.

Thank you very much, Hubble!

SierraKomodo reviewed (changes requested):
Some things I noticed while testing locally:

 - [ ] Catfish's helm and sensors console are connected to Verne, not the shuttle
 - [ ] Airlocks should have `name` vars set to be more descriptive of what room each one connects to - Similar to how the Torch labels them
 - [ ] Catfish has no fuel port and no fuel to conduct z-level transitions with
 - [ ] Turret probably should start on stun, not lethal
 - [ ] Gas mixer in engine room starts unconfigured and disconnected
 - [ ] Inconsistant striping colors - 
![dreamseeker_udLm7KeatC](https://user-images.githubusercontent.com/11140088/91492217-200b2480-e86a-11ea-8df2-075731c32f6e.png)

 - [ ] Catfish's inner airlock access button doesn't work
 - [ ] Gas pump on the Catfish shouldn't be under a chair - It's difficult to see and click on

SierraKomodo commented:
Also, merge conflict

Albens commented:
> * [x]  Airlocks should have `name` vars set to be more descriptive of what room each one connects to - Similar to how the Torch labels them
> * [x]  Catfish has no fuel port and no fuel to conduct z-level transitions with
> * [x]  Gas mixer in engine room starts unconfigured and disconnected
> * [x]  Catfish's inner airlock access button doesn't work
> * [x]  Gas pump on the Catfish shouldn't be under a chair - It's difficult to see and click on
> Also, merge conflict

These issues addressed.

> * [ ]  Catfish's helm and sensors console are connected to Verne, not the shuttle
> * [ ]  Turret probably should start on stun, not lethal
> * [ ]  Inconsistant striping colors

The Helm/Sensors might be a Torch bug? I'm not even sure how to cause/ fix that. In my testing I never had it. 
Will relook.

Turrets, as discussed, are an Anti-boarding tool, designed to disincentive raiding prior to the ship awakening. Until other means exist, it's the most applicable and effective tool. Stun just.. doesn't do anything, as there's no risk. 
Lethals mostly just deters as is, as in testing, players just scrambled up and around the turrets using Z-levels, and then breach in via r-wall. 
Honestly. If the turrets and blast-doors are up, that means no one has awoken.

Striping is a bug and should be bug reported (by me I guess)

Albens commented:
Okay so; 
Loadout defines done.
Testing added. (sorta).
Had to fix a Vox bug not having desc causing travis errors.

Issues: Shuttle, maybe the landing might still not work. Need help pls. 
Gun isn't in yet and might not be. Need help pls.
Gotta configure power.

SierraKomodo reviewed (dismissed):
Runtimes (These seem to be triggering in testing even if the Verne isn't being loaded):
```
[07:00:00] Runtime in shuttle.dm,52: Shuttle "SRV Venerable Catfish" could not find its starting location.
  proc name: New (/datum/shuttle/New)
  src: SRV Venerable Catfish (/datum/shuttle/autodock/overmap/verne)
  call stack:
  SRV Venerable Catfish (/datum/shuttle/autodock/overmap/verne): New(null, null)
  SRV Venerable Catfish (/datum/shuttle/autodock/overmap/verne): New(null, null)
  SRV Venerable Catfish (/datum/shuttle/autodock/overmap/verne): New(null, null)
  Shuttle (/datum/controller/subsystem/shuttle): initialize shuttle(/datum/shuttle/autodock/overma... (/datum/shuttle/autodock/overmap/verne))
  Shuttle (/datum/controller/subsystem/shuttle): initialize shuttle(/datum/shuttle/autodock/overma... (/datum/shuttle/autodock/overmap/verne))
  Shuttle (/datum/controller/subsystem/shuttle): initialize shuttles()
  Shuttle (/datum/controller/subsystem/shuttle): clear init queue()
  Shuttle (/datum/controller/subsystem/shuttle): Initialize(657167)
  Shuttle (/datum/controller/subsystem/shuttle): DoInitialize(657167)
  Master (/datum/controller/master): Initialize(10, 0)
```
```
[07:00:00] Runtime in shuttle_autodock.dm,25: Cannot read null.docking_controller
  proc name: New (/datum/shuttle/autodock/New)
  src: SRV Venerable Catfish (/datum/shuttle/autodock/overmap/verne)
  call stack:
  SRV Venerable Catfish (/datum/shuttle/autodock/overmap/verne): New(null, null)
  SRV Venerable Catfish (/datum/shuttle/autodock/overmap/verne): New(null, null)
  Shuttle (/datum/controller/subsystem/shuttle): initialize shuttle(/datum/shuttle/autodock/overma... (/datum/shuttle/autodock/overmap/verne))
  Shuttle (/datum/controller/subsystem/shuttle): initialize shuttle(/datum/shuttle/autodock/overma... (/datum/shuttle/autodock/overmap/verne))
  Shuttle (/datum/controller/subsystem/shuttle): initialize shuttles()
  Shuttle (/datum/controller/subsystem/shuttle): clear init queue()
  Shuttle (/datum/controller/subsystem/shuttle): Initialize(657167)
  Shuttle (/datum/controller/subsystem/shuttle): DoInitialize(657167)
  Master (/datum/controller/master): Initialize(10, 0)
```

Raptor1628 commented:
Neat

Hubblenaut commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,389 @@
+	vessel_mass = 5000
+	max_speed = 1/(3 SECONDS)
+	fore_dir = NORTH
+	initial_restricted_waypoints = list(
```
```suggestion
	initial_restricted_waypoints = list(
		"SRV Venerable Catfish" = list("nav_hangar_verne", "nav_verne_4")
	)
```
The way you defined the list right now, the second entry in the list initialization would immediately overwrite the same key, thus effectively removing the hangar from the list of restricted waypoints.

Hubblenaut commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,389 @@
+	initial_generic_waypoints = list(
+		"nav_verne_1",
+		"nav_verne_2",
+		"nav_verne_3",
```
Rogue comma at end of list.

Hubblenaut commented on maps/away/verne/verne_shuttles.dm:
```diff
@@ -0,0 +1,86 @@
+	defer_initialisation = TRUE
+
+/obj/effect/shuttle_landmark/verne/dock
+	name = "SRV Venerable Catfish Dock"
```
`SRV Venerable Catfish Hangar` is probably more accurate.

Albens commented:
All that should be left:
Job stats.
Reactor settings.

Albens commented:
Need @Spookerton lore approval if IPC CTI is validsalid.

Albens commented:
What's left:
Check the reactor works and lasts as advertised.
Lore.
Check the SM computers are hooked up properly.

Albens commented:
.. It's done.
100% playable.
Lore documents remain. 
@Rowtree will be the one doing them, thankfully. I suck at it.

SierraKomodo reviewed (changes requested):
The access buttons for the venerable catfish appear to not work.

Albens commented:
Updated as per @SierraKomodo comment.
While I was at it I went and found various freq issues, and fixed them too.
And then some other minor stuff, leaks, clothing/belts, pipes, such and such.

SierraKomodo reviewed (changes requested):
- Supermatter Research area, south airlock has a weird setup - Why are you using blast doors instead of airlock doors? Why is there an inner chamber and airlock door that's not being used as part of the airlock?
- Supermatter Research area, south airlock, the blast doors don't open when cycling and don't seem to have a way to open them
- Supermatter Research area, west airlock lacks any means to cycle/control the doors
- The various protective blast doors on the exterior portions and the button labelled 'Window Blast Doors' - It isn't readily apparent these are connected.
- The omni gas filter south of the west phoron tanks on the lower level isn't configured
- Verne Thrusters area, SE airlock doesn't have an exterior access button
- Telecommunications chamber has no vent or scrubber
- Secure holding doesn't seem to have a method of unbolting without hacking it
- Verne professor's office lacks a fire alarm

Albens commented:
> *Supermatter Research area, south airlock has a weird setup - Why are you using blast doors instead of airlock doors? Why is there an inner chamber and airlock door that's not being used as part of the airlock?
- * Supermatter Research area, south airlock, the blast doors don't open when cycling and don't seem to have a way to open them
- * Supermatter Research area, west airlock lacks any means to cycle/control the doors

https://gyazo.com/74d9b114fc0fd93926ac7358a07ec75a
The blastdoors are the d3 Lockdown doors. They are tied to buttons in both the Fore and Aft Stack. Blue highlight for reference.
They're here for the "prevent pre-spawn raiding" etc.
Inner chamber is actually a radiation shield; Orange Wall, door, rad locker and suit storage unit are there to allow you to cycle to exterior to handle gas transfer, and then cycle back in and change suit.
This whole area is a biittt tight yea. Thankfully it's mostly ornamental.

https://gyazo.com/09265e119dcbc099460cffc6499e5836
These two areas are just access rooms, like our SM does. They are there to shield from rads.

- * The various protective blast doors on the exterior portions and the button labelled 'Window Blast Doors' - It isn't readily apparent these are connected.

I don't follow- All the Verne Protective Blast Doors are controlled by a button on the Bridge (for d1 and d2)
Or in the Fore/Aft stack for d3.
The buttons are named the exact same as the doors they control.

- * The omni gas filter south of the west phoron tanks on the lower level isn't configured

Yea I can't get that one to configure. There's likely a bug but I'm not sure. The tag should be correct. I'll take another peak.
- * Verne Thrusters area, SE airlock doesn't have an exterior access button
https://gyazo.com/8cdfc86594c2c32b16dd0e3c23b9c429

- [ ]  Telecommunications chamber has no vent or scrubber

I'll add some once I'm home from work/soon!
- * Secure holding doesn't seem to have a method of unbolting without hacking it
- * Verne professor's office lacks a fire alarm

https://gyazo.com/029dd4b44d48334858b9ea6a2e33eeeb
Both already in!

Albens commented:
Updated. Yeep.

Removed the PCC's, the sprite sucked. Such is life.
Transit for the Catfish was added, forgot about that.
Fixed the mis-configured filter.
Changed about the bridge turrets due to them.. glitching out about carp in specific locations.
Added torch-docking location.
Might've updated other things idk, small stuff.

Albens commented:
Tentative Ready for Review/Merge.

If any reviewer wishes me to host the map and explain anything to them, I am willing and open.

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+	crew_jobs = list(
+		/datum/job/submap/CTI_pilot,
+		/datum/job/submap/CTI_engineer,
+		/datum/job/submap/CTI_Undergraduate_Xenoscience_Researcher
```
```suggestion
		/datum/job/submap/CTI_Undergraduate_Xenoscience_Researcher,
```

SierraKomodo commented on code/datums/supplypacks/supply.dm:
```diff
@@ -78,7 +78,8 @@
 					/obj/item/weapon/storage/box/cola/grapejuice,
-					/obj/item/weapon/storage/box/cola/sodawater)
+					/obj/item/weapon/storage/box/cola/sodawater,
+					/obj/item/weapon/storage/box/cola/beastenergy)
```
```suggestion
					/obj/item/weapon/storage/box/cola/beastenergy,
				)
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+	initial_generic_waypoints = list(
+		"nav_verne_1",
+		"nav_verne_2",
+		"nav_verne_3"
```
```suggestion
		"nav_verne_3",
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+	area_usage_test_exempted_root_areas = list(/area/verne)
+	shuttles_to_initialise = list(
+		/datum/shuttle/autodock/overmap/verne,
+		/datum/shuttle/autodock/ferry/verne
```
```suggestion
		/datum/shuttle/autodock/ferry/verne,
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+		/area/verne/engineering/powergen = NO_SCRUBBER|NO_VENT,
+		/area/verne/engineering/thrusters = NO_SCRUBBER|NO_VENT,
+		/area/verne/catfish/engineering = NO_SCRUBBER|NO_VENT,
+		/area/verne/lift = NO_SCRUBBER|NO_VENT|NO_APC
```
```suggestion
		/area/verne/lift = NO_SCRUBBER|NO_VENT|NO_APC,
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+		/area/verne/lift = NO_SCRUBBER|NO_VENT|NO_APC
+	)
+	area_coherency_test_subarea_count = list(
+		/area/verne/engineering/external = 20
```
```suggestion
		/area/verne/engineering/external = 20,
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+
+/datum/reagent/toxin/phoron/safe
+	name = "tericadone"
+	description = "Phoron substitute currently in labratory testing"
```
```suggestion
	description = "Phoron substitute currently in laboratory testing"
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+	radiation_release_modifier = 1
+
+/obj/structure/closet/crate/secure/large/phoron/experimentalsm
+	name = "experimental SM crate"
```
Feels inconsistent that everything else is named `supermatter` but this is named `SM`
```suggestion
	name = "experimental supermatter crate"
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+	required_reagents = list(/datum/reagent/cryoxadone = 1, /datum/reagent/sodium = 1, /datum/reagent/toxin/phoron/safe = 0.1)
+
+/obj/machinery/suit_storage_unit/engineering/verne
+	req_access = (access_verne)
```
This syntax confuses me somewhat - Shouldn't this be `list(access_verne)` or does beyond have support for shorthand lists that I'm not aware of?

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+	req_access = (access_verne)
+
+/obj/machinery/suit_storage_unit/science/verne
+	req_access = (access_verne)
```
Same as above about `list(access_verne)`

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+/obj/machinery/power/smes/buildable/preset/verne
+	uncreated_component_parts = list(
+		/obj/item/weapon/stock_parts/smes_coil/super_io = 2,
+		/obj/item/weapon/stock_parts/smes_coil/super_capacity = 2)
```
```suggestion
		/obj/item/weapon/stock_parts/smes_coil/super_capacity = 2,
	)
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+/obj/machinery/power/smes/buildable/preset/verne/shuttle
+	uncreated_component_parts = list(
+		/obj/item/weapon/stock_parts/smes_coil/super_io = 1,
+		/obj/item/weapon/stock_parts/smes_coil/super_capacity = 1)
```
```suggestion
		/obj/item/weapon/stock_parts/smes_coil/super_capacity = 1,
	)
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+
+/obj/item/weapon/paper/verne
+	name = "power usage"
+	info = "Verne Pilots and Engineers, remember to turn off the thrusters when you're done with maneuvers.\
+	<BR>Power useage settles at 137 kW with them off, just under the ICRER-2 150 kW standard operation range.\
+	<BR>When you're performing maneuvers, put the ICRER-2 up to 180 kW, turn the gas cooler on, and remember to feed it Coolant!\
+	<BR>After you're done maneuvers or if an hour has passed, drop it down to 150 kW and turn the gas cooler off. \
+	<BR>Otherwise it'll explode. \
+	<BR>If it explodes, you'll totally destroy the aft Engineering Stack, stranding us. \
+	<BR><b>Your follow up to this matter is required and appreciated.</b>"
```
Grammar/spelling

```suggestion
	info = "Verne Pilots and Engineers, remember to turn off the thrusters when you're done with maneuvers.\
	<BR>Power usage settles at 137 kW with them off, just under the ICRER-2 150 kW standard operating range.\
	<BR>When you're performing maneuvers, put the ICRER-2 up to 180 kW, turn the gas cooler on, and remember to feed it Coolant!\
	<BR>After you're done with maneuvers or if an hour has passed, drop it down to 150 kW and turn the gas cooler off. \
	<BR>Otherwise it'll explode. \
	<BR>If it explodes, you'll totally destroy the aft Engineering Stack, stranding us. \
	<BR><b>Your follow up to this matter is required and appreciated.</b>"
```

SierraKomodo commented on maps/away/verne/verne.dm:
```diff
@@ -0,0 +1,275 @@
+
+/obj/item/weapon/paper/verne/briefing
+	name = "Survey Team Cyrenica briefing"
+	info = "Good waking, Survey Team Cyrenica.<BR>\
+		Dr. Schoonhoven and I have been negotiating the routing we intended to take the SRV Verne for this expedition.<BR>\
+		This matter was improperly discussed during the inital expediton planning, and I promise, we will be making amends upon our return to Tau-Ceti.\
+		After your last shift, the executive decision was made to put the SRV Verne onto extended automation.<BR>\
+		You will be alone during the intial day of your shift. <BR>\
+		The routing Dr. Schoonhoven and I negotiated will take us further than every before. This comes with consequences; I trust you will be careful during your Survey missions. <BR>\
+		We will be making history with this expediton. You will have earned the honor of having chased the God of the Sun.<BR>\
+		I have scheduled support staff to awake during the second day of your shift. They will be awake for 48 hours, to provide overlapping coverage to Survey Team Theta, after you have rentered stasis.<BR>\
+		Your scheduled support staff are;<BR>\
+		Physician Dr. Milli Newman<BR>\
+		Non-Essential Systems Technician Neel Barrett.<BR>\
+		They will awake 24 hours after your shift begins.<BR>\
+		Your shift will last for the standard 48 hours.<BR>\
+		Your duties remain unchanged;<BR>\
+		Survey the Exoplanet you have been brought to.<BR>\
+		The Students will be performing their studies as required.<BR>\
+		If you feel any nearby sites are of value, you have my permission to divert to them.<BR>\
+		Given our tenuous staffing, it comes with my highest recommendations that you are aware of any other active vessels in your system. If they are amenable to working with you, or to assisting you if required. <BR>\
+		I firmly request you do not touch the Supermatter Testing facilities, as I have scheduled later classes to awake to perform their studies using the stock we have remaining.<BR>\
+		Our supplies remain at acceptable levels at this time, but I will permit barter or trading if required.\
+		Do not sell the resources your fellow students will require.<BR>\
+		I await your results.<BR>\
+		Dr. Andre Richardson"
```
Grammar/spelling

```suggestion
	info = "Good waking, Survey Team Cyrenica.<BR>\
		Dr. Schoonhoven and I have been negotiating the routing we intended to take the SRV Verne for this expedition.<BR>\
		This matter was improperly discussed during the initial expedition planning, and I promise, we will be making amends upon our return to Tau-Ceti.\
		After your last shift, the executive decision was made to put the SRV Verne onto extended automation.<BR>\
		You will be alone during the initial day of your shift. <BR>\
		The routing Dr. Schoonhoven and I negotiated will take us further than ever before. This comes with consequences; I trust you will be careful during your Survey missions. <BR>\
		We will be making history with this expedition. You will have earned the honor of having chased the God of the Sun.<BR>\
		I have scheduled support staff to awake during the second day of your shift. They will be awake for 48 hours, to provide overlapping coverage to Survey Team Theta, after you have entered stasis.<BR>\
		Your scheduled support staff are;<BR>\
		Physician Dr. Milli Newman<BR>\
		Non-Essential Systems Technician Neel Barrett.<BR>\
		They will awake 24 hours after your shift begins.<BR>\
		Your shift will last for the standard 48 hours.<BR>\
		Your duties remain unchanged;<BR>\
		Survey the Exoplanet you have been brought to.<BR>\
		The students will be performing their studies as required.<BR>\
		If you feel any nearby sites are of value, you have my permission to divert to them.<BR>\
		Given our tenuous staffing, it comes with my highest recommendations that you are aware of any other active vessels in your system if they are amenable to working with you or to assist you if required. <BR>\
		I firmly request you do not touch the Supermatter Testing facilities, as I have scheduled later classes to awake to perform their studies using the stock we have remaining.<BR>\
		Our supplies remain at acceptable levels at this time, but I will permit barter or trading if required.\
		Do not sell the resources your fellow students will require.<BR>\
		I await your results.<BR>\
		Dr. Andre Richardson"
```

SierraKomodo commented on maps/away/verne/verne_jobs.dm:
```diff
@@ -0,0 +1,107 @@
+	total_positions = 1
+	outfit_type = /decl/hierarchy/outfit/job/verne/pilot
+	supervisors = "the CTI Professor in Command of the Verne"
+	info = "You are an employee on the SRV Verne, as it carries the students of the presitigous Ceti Technical Institute conducting research \
+	in the depths of space. Your job on the Survey team is simple; pilot the SRV Verne and the SRV Venerable Catfish; protect the students; and assist in their studies. Your Survey team has awoken \
+ 	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
```
Spelling

```suggestion
	info = "You are an employee on the SRV Verne, as it carries the students of the prestigious Ceti Technical Institute conducting research \
	in the depths of space. Your job on the Survey team is simple; pilot the SRV Verne and the SRV Venerable Catfish; protect the students; and assist in their studies. Your Survey team has awoken \
 	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
```

SierraKomodo commented on maps/away/verne/verne_jobs.dm:
```diff
@@ -0,0 +1,107 @@
+ 	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
+	whitelisted_species = list(SPECIES_HUMAN,SPECIES_IPC,SPECIES_SPACER,SPECIES_GRAVWORLDER,SPECIES_VATGROWN,SPECIES_BOOSTER,SPECIES_TRITONIAN)
+	required_language = LANGUAGE_HUMAN_EURO
+	min_skill = list(	SKILL_EVA   = SKILL_BASIC,
+						SKILL_WEAPONS     = SKILL_ADEPT,
+						SKILL_PILOT = SKILL_ADEPT)
+
+	max_skill = list(   SKILL_PILOT       = SKILL_MAX,
+	                    SKILL_SCIENCE     = SKILL_MAX)
```
The suggested changes look a lot cleaner.

```suggestion
	min_skill = list(
		SKILL_EVA     = SKILL_BASIC,
		SKILL_WEAPONS = SKILL_ADEPT,
		SKILL_PILOT   = SKILL_ADEPT,
	)

	max_skill = list(
		SKILL_PILOT   = SKILL_MAX,
		SKILL_SCIENCE = SKILL_MAX,
	)
```

SierraKomodo commented on maps/away/verne/verne_jobs.dm:
```diff
@@ -0,0 +1,107 @@
+	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
+	whitelisted_species = list(SPECIES_HUMAN,SPECIES_IPC,SPECIES_SPACER,SPECIES_GRAVWORLDER,SPECIES_VATGROWN,SPECIES_BOOSTER,SPECIES_TRITONIAN)
+	required_language = LANGUAGE_HUMAN_EURO
+	min_skill = list(   SKILL_COMPUTER     = SKILL_BASIC,
+	                    SKILL_EVA          = SKILL_BASIC,
+	                    SKILL_CONSTRUCTION = SKILL_ADEPT,
+	                    SKILL_ELECTRICAL   = SKILL_BASIC,
+	                    SKILL_ATMOS        = SKILL_BASIC,
+	                    SKILL_ENGINES      = SKILL_BASIC)
+
+	max_skill = list(   SKILL_CONSTRUCTION = SKILL_MAX,
+	                    SKILL_ELECTRICAL   = SKILL_MAX,
+	                    SKILL_ATMOS        = SKILL_MAX,
+	                    SKILL_ENGINES      = SKILL_MAX)
```
```suggestion
	min_skill = list(
		SKILL_COMPUTER     = SKILL_BASIC,
		SKILL_EVA          = SKILL_BASIC,
		SKILL_CONSTRUCTION = SKILL_ADEPT,
		SKILL_ELECTRICAL   = SKILL_BASIC,
		SKILL_ATMOS        = SKILL_BASIC,
		SKILL_ENGINES      = SKILL_BASIC,
	)

	max_skill = list(
		SKILL_CONSTRUCTION = SKILL_MAX,
		SKILL_ELECTRICAL   = SKILL_MAX,
		SKILL_ATMOS        = SKILL_MAX,
		SKILL_ENGINES      = SKILL_MAX,
	)
```

SierraKomodo commented on maps/away/verne/verne_jobs.dm:
```diff
@@ -0,0 +1,107 @@
+	total_positions = 1
+	outfit_type = /decl/hierarchy/outfit/job/verne/engineer
+	supervisors = "the CTI Professor in Command of the Verne"
+	info = "You are an employee on the SRV Verne, as it carries the students of the presitigous Ceti Technical Institute conducting research \
+ 	in the depths of space. Your job on the Survey team is simple; handle engineering work on SRV Verne and the SRV Venerable Catfish; protect the students; and assist in their studies. Your Survey team has awoken \
+	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
```
Grammar/spelling

```suggestion
	info = "You are an employee on the SRV Verne, as it carries the students of the prestigious Ceti Technical Institute conducting research \
 	in the depths of space. Your job on the Survey team is simple: Handle engineering work on the SRV Verne and the SRV Venerable Catfish, protect the students, and assist in their studies. Your Survey team has awoken \
	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
```

SierraKomodo commented on maps/away/verne/verne_jobs.dm:
```diff
@@ -0,0 +1,107 @@
+	supervisors = "the CTI Professor in Command of the Verne"
+	total_positions = 2
+	outfit_type = /decl/hierarchy/outfit/job/verne/researcher
+	info = "You are an undergraduate xenoscience researcher on the SRV Verne, alongside the rest of your class of the presitigous Ceti Technical Institute conducting research \
+	in the depths of space. A survey team will be accompanying you, on hand to assist your studies on the exoplanets in this system. Your team has awoken \
+	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
```
Spelling

```suggestion
	info = "You are an undergraduate xenoscience researcher on the SRV Verne, alongside the rest of your class of the prestigious Ceti Technical Institute conducting research \
	in the depths of space. A survey team will be accompanying you, on hand to assist your studies on the exoplanets in this system. Your team has awoken \
	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
```

SierraKomodo commented on maps/away/verne/verne_jobs.dm:
```diff
@@ -0,0 +1,107 @@
+	to find the Verne running at low capacity, under-staffed, with much of the automated life support systems doing the heavy lifting."
+	whitelisted_species = list(SPECIES_HUMAN,SPECIES_SPACER,SPECIES_GRAVWORLDER,SPECIES_VATGROWN,SPECIES_BOOSTER,SPECIES_TRITONIAN)
+	required_language = LANGUAGE_HUMAN_EURO
+	min_skill = list(   SKILL_BUREAUCRACY = SKILL_BASIC,
+	                    SKILL_COMPUTER    = SKILL_BASIC,
+	                    SKILL_DEVICES     = SKILL_BASIC,
+	                    SKILL_SCIENCE     = SKILL_ADEPT)
+
+	max_skill = list(   SKILL_ANATOMY     = SKILL_MAX,
+	                    SKILL_DEVICES     = SKILL_MAX,
+	                    SKILL_SCIENCE     = SKILL_MAX)
```
```suggestion
	min_skill = list(
		SKILL_BUREAUCRACY = SKILL_BASIC,
		SKILL_COMPUTER    = SKILL_BASIC,
		SKILL_DEVICES     = SKILL_BASIC,
		SKILL_SCIENCE     = SKILL_ADEPT,
	)

	max_skill = list(
		SKILL_ANATOMY = SKILL_MAX,
		SKILL_DEVICES = SKILL_MAX,
		SKILL_SCIENCE = SKILL_MAX,
	)
```

SierraKomodo commented on maps/away/verne/verne_shuttles.dm:
```diff
@@ -0,0 +1,96 @@
+/datum/shuttle/autodock/overmap/verne
+	name = "SRV Venerable Catfish"
+	move_time = 90
+	shuttle_area = list(/area/verne/catfish, /area/verne/catfish/engineering)
```
```suggestion
	shuttle_area = list(
		/area/verne/catfish,
		/area/verne/catfish/engineering,
	)
```

SierraKomodo commented on maps/torch/torch_overmap.dm:
```diff
@@ -15,7 +15,8 @@
 		"ITV Vulcan" = list("nav_hangar_gantry_torch_two"),
-		"ITV Spiritus" = list("nav_hangar_gantry_torch_three")
+		"ITV Spiritus" = list("nav_hangar_gantry_torch_three"),
+		"SRV Venerable Catfish" = list("nav_verne_5") //docking location for verne shuttle
```
```suggestion
		"SRV Venerable Catfish" = list("nav_verne_5"), //docking location for verne shuttle
```

SierraKomodo commented on maps/torch/torch_overmap.dm:
```diff
@@ -73,7 +74,8 @@
 		"nav_skrellscout_dock",
-		"nav_ert_dock"
+		"nav_ert_dock",
+		"nav_verne_5"
```
```suggestion
		"nav_verne_5",
```

SierraKomodo reviewed (changes requested):
Mostly code styling and grammar/spelling stuff.

SierraKomodo reviewed (commented):
In `verne1.dmm`, for the southern airlock, the western door doesn't have an access button - Intended?
![StrongDMM_INTuZMN38E](https://user-images.githubusercontent.com/11140088/93915424-da9d1400-fcbc-11ea-9d7e-d961e47ed3f2.png)

Albens committed:
Verne commit

With much help from Venku22, Nerezza.
Apply suggestions from code review

Co-authored-by: Sierra Brown (SierraKomodo) <SierraKomodo@users.noreply.github.com>

Albens commented:
Updated to address all comments.
